code-challenges/array-binary-search/array-binary-search.test.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const binarySearch = require('./array-binary-search.js');
+
+describe('Testing binary search code challenges, A', () => {
+  it('Should reverse an array', () => {
```

Comment:
I would change this it to something like 'successfully find the key in the array'